DynamicPackagingVODContent/StreamHLSAndDASH/src/main/java/sample/StreamHLSAndDASH.java

```diff
@@ -358,22 +394,23 @@ private static StreamingLocator getStreamingLocator(MediaManager manager, String
      * Checks if the streaming endpoint is in the running state, if not, starts it.
      *
      * @param manager           The entry point of Azure Media resource management.
-     * @param resourceGroup     The name of the resource group within the Azure subscription.
+     * @param resourceGroup     The name of the resource group within the Azure
+     *                          subscription.
      * @param accountName       The Media Services account name.
      * @param locatorName       The name of the StreamingLocator that was created.
      * @param streamingEndpoint The streaming endpoint.
      * @return List of streaming urls.
      */
-    private static List<String> getHlsAndDashStreamingUrls(MediaManager manager, String resourceGroup, String accountName,
-                                                           String locatorName, StreamingEndpoint streamingEndpoint) {
+    private static List<String> getHlsAndDashStreamingUrls(MediaServicesManager manager, String resourceGroup,
+            String accountName,
+            String locatorName, StreamingEndpoint streamingEndpoint) {
         List<String> streamingUrls = new ArrayList<>();
-        ListPathsResponse paths = manager.streamingLocators().listPathsAsync(resourceGroup, accountName, locatorName)
-                .toBlocking().first();
+        ListPathsResponse paths = manager.streamingLocators().listPaths(resourceGroup, accountName, locatorName);
 
         for (StreamingPath path : paths.streamingPaths()) {
```

Comment:
I dont really like making the streaming URLs this way.  I prefer to build them a bit more explicit for CMAF as we do in the Typescript samples. 

This one spits out non CMAF and older CSF paths which are not what we want to show.